.env.example

```diff
@@ -19,4 +19,9 @@ STRAPI_LOG_FORCE_COLOR = true
 STRAPI_LOG_PRETTY_PRINT = true
 NODE_ENV = 'development'
 BROWSER = true
-ENV_PATH = './.env'
\ No newline at end of file
+ENV_PATH = './.env'
+
+# Publish changes plugin
+GITHUB_APP_KEY = abcde
```

Comment:
This row should clarify that we expect a PEM RSA key starting with `-----BEGIN RSA PRIVATE KEY-----\n`...

Not sure how one would put that in an env file, though.

Reply:
I completely removed this from `.env.example` since it will never be passed there. Instead I mention about this environmental variable in `README.md`